fudge-ratio/src/main/java/com/google/sps/data/Activity.java

```diff
@@ -0,0 +1,19 @@
+package com.google.sps.data;
+import java.util.ArrayList; 
+
+public final class Activity {
+
+    private final long id; //Activity unique id
+    private final String name; //Activity name
+    private final String type;
+    private final String expectedDuration; //Expected time to complete activity
```

Comment:
shouldn't we also be storing the actual time used so that we can calculate the ratio